scripts/pre_install.sh

```
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+PROJECT_ROOT="/home/ec2-user/app"
+
+cd $PROJECT_ROOT
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)